packages/react-components/src/components/VideoGallery/DefaultLayout.tsx

```diff
@@ -66,7 +66,10 @@ export const DefaultLayout = (props: DefaultLayoutProps): JSX.Element => {
     );
   });
 
-  const [indexesToRender, setIndexesToRender] = useState<number[]>([0]);
+  /** instantiate indexes available to render with indexes available that would be on first page */
+  const [indexesToRender, setIndexesToRender] = useState<number[]>([
+    ...Array(maxRemoteVideoStreams - activeVideoStreams).keys()
```

Comment:
So this is the key to fix the bug, nice!

A small note here: For some interesting components which doesn't strictly follow the order of the array, we might re-render the initial tiles -> dispose them -> create new tiles, we need to take care of this case when those components are here

let's add this description into comment, so someone will handle this part well when developing "not follow order" component

Reply:
I dont think this will be affected by order no? this array is strictly telling the render function which which indexes to render, so the tiles that are present on the first page shouldn't matter and then should the page change this array will be updated.

Reply:
I dont think this will be affected by order no? this array is strictly telling the render function which which indexes to render, so the tiles that are present on the first page shouldn't matter and then should the page change this array will be updated.